cmd/image-builder/logging.go

```diff
@@ -38,13 +29,12 @@ func (h *ctxHook) Fire(e *logrus.Entry) error {
 		return nil
 	}
 
-	e.Data["request_id"] = e.Context.Value(requestIdCtx)
-	e.Data["insights_id"] = e.Context.Value(insightsRequestIdCtx)
+	e.Data["request_id"] = common.RequestId(e.Context)
+	e.Data["insights_id"] = common.InsightsRequestId(e.Context)
 
-	if rd, ok := e.Context.Value(requestDataCtx).(logrus.Fields); ok {
-		for k, v := range rd {
-			e.Data[k] = v
-		}
+	rd := common.RequestData(e.Context)
+	for k, v := range rd {
```

Comment:
will this work in case the context value of `RequestData` was not set and `rd == nil`?

Reply:
Good catch, it is alwas set but not returning nil solves it nicely.